tests/model/test_model.py

```diff
@@ -2079,20 +2101,60 @@ def test_toggle_stream_pinned_status(
                     "id": 0,
                 },
                 True,
+                False,
+                False,
+                id="op_stop",
             ),
         ],
-        ids=["not_in_pm_narrow", "not_in_pm_narrow_with_sender", "start", "stop"],
     )
-    def test__handle_typing_event(self, mocker, model, narrow, event, called):
+    def test__handle_typing_event(
+        self,
+        mocker,
+        model,
+        narrow,
+        event,
+        is_animation_in_progress,
+        expected_is_recipient_typing,
+        expected__show_typing_animation,
+    ):
         event["type"] = "typing"
 
-        mocker.patch("zulipterminal.ui.View.set_footer_text")
+        mocker.patch("zulipterminal.model.Model._show_typing_animation")
```

Comment:
You can just use `MODEL + "._show_typing_animation"` here. The `MODEL` constant is defined at the top of the file as `zulipterminal.model.Model`. This will keep your patch consistent with the other tests in the file.